src/omv/modules/py_tf.c

```diff
@@ -265,41 +265,83 @@ STATIC py_tf_model_obj_t *py_tf_load_alloc(mp_obj_t path_obj) {
     }
 }
 
+typedef enum {
+    PY_TF_SCALE_NONE,
+    PY_TF_SCALE_0_1,
+    PY_TF_SCALE_N1_1,
+    PY_TF_SCALE_N128_127
+} py_tf_input_scale_t;
+
+typedef enum {
+    PY_TF_R_MEAN,
+    PY_TF_G_MEAN,
+    PY_TF_B_MEAN,
+    PY_TF_R_STDDEV,
+    PY_TF_G_STDDEV,
+    PY_TF_B_STDDEV,
+    PY_TF_INPUT_NORMALIZATION_SIZE
+} py_tf_input_normalization_t;
```

Comment:
```suggestion
typedef enum {
    PY_TF_R_MEAN,
    PY_TF_G_MEAN,
    PY_TF_B_MEAN,
    PY_TF_R_STDDEV,
    PY_TF_G_STDDEV,
    PY_TF_B_STDDEV,
    PY_TF_INPUT_STATS_SIZE
} py_tf_input_stats_t;

A better, shorter, name for this is `input_stats` which they are.
```

Reply:
This is an outdated comment. I changed it previously to what was discussed in the meeting with mean/std.